account_financial_report_qweb/__openerp__.py

```diff
@@ -35,8 +35,11 @@
         'report/templates/trial_balance.xml',
         'view/account_view.xml'
     ],
-    'test': [
-    ],
+    'external_dependencies': {
+        "python": [
+            "xlsxwriter",
+        ],
+    },
```

Comment:
I updated the patch and put the try lower in the stack. Please review that, you probably missed that.